Changelog.md

```diff
@@ -6,8 +6,7 @@ Language Features:
 Compiler Features:
  * Commandline Interface: Model checker option ``--model-checker-targets`` also accepts ``outOfBounds``.
  * Low-Level Inliner: Inline ordinary jumps to small blocks and jumps to small blocks that terminate.
- * SMTChecker: Report out of bounds index access for arrays and fixed bytes.
- * Standard JSON: Model checker option ``settings.modelChecker.targets`` also accepts ``outOfBounds``.
```

Comment:
Why are you removing the changelog entries from your last PR?

Reply:
Fixed :grimacing: